nevergrad/optimization/experimentalvariants.py

```diff
@@ -302,6 +302,9 @@
 ProgONOPO13 = ConfSplitOptimizer(num_optims=13, progressive=True, multivariate_optimizer=OptimisticNoisyOnePlusOne).set_name(
     "ProgONOPO13", register=True
 )
+ProgONOPOInf = ConfSplitOptimizer(num_optims=10000, progressive=True, multivariate_optimizer=OptimisticNoisyOnePlusOne).set_name(
```

Comment:
define in an optgroup?

Reply:
Done.

Reply:
Done.

Reply:
No, what I mean is that this should be directly defined in the opt group and not here.
In the optgruop you can just replace the string "ProgONOPOInf" by `ConfSplitOptimizer(num_optims=10000, progressive=True, multivariate_optimizer=OptimisticNoisyOnePlusOne).set_name("ProgONOPOInf")`

If it is used only once, then it is preferably not to define it in experimentalvariants